### ISSUE TITLE
Hiding some hours in the calendar causes the Google Calendar website to jitter for a few s after I load it

### ISSUE THREAD

**Issue by ars4l4n** [GCalToolkit/GCalPlus#4]
When I'm using GCalPlus's feature to only display certain hours in the calendar, the website always jitters for a few seconds after loading it, causing me to misclick elements. Can you fix that?

![2024-07-22_20-10-25](https://github.com/user-attachments/assets/60cb577d-9f8f-4e49-8e82-f6928a10aba1)
[Video of the Issue](https://github.com/user-attachments/assets/42e2be17-8881-485f-9975-38a8197db54b
)

I reproduced the issue on the following configurations:
-24" monitor, Windows 10 Pro, 64 bit 22H2 (OS Build 19045.4651), OS Scaling: 100%, Browser Zoom: 100%, Brave Browser (Version 1.67.134 Chromium: 126.0.6478.186 (Official Build) (64-bit)), no other extensions or GCalPlus features enabled
-same on Chrome but with other extensions enabled
-12,3" display, Windows 11 Home, 23H2, OS Scaling: 200% (recommended), Browser Zoom: 100%, Brave Browser (Version 1.67.134 Chromium: 126.0.6478.186 (Official Build) (64-bit)), no other extensions or GCalPlus features enabled
-same on Chrome but with other extensions enabled

**Comment by GCalToolkit** [comment]
Several of GCalPlus's options can cause jitters because of the way Google randomly updates the UI, which GCalPlus then has to correct again - but I haven't ever noticed this myself with the option to hide certain hours. Have you tried using only integers, or using 7.5 instead of '7.30'?

**Comment by ars4l4n** [comment]
@GCalToolkit Yes, I've just tried both these things and the issue remains. I haven't even enabled any other features of the extension. Tried this in the Brave browser with no extensions enabled; same result.

Do you think a non-default display scaling level could have an effect on this? Mine is set to 111%. I wouldn't change it though. Would you make a workaround if that was the issue?

**Comment by GCalToolkit** [comment]
Unfortunately changing the UI in Google Calendar is quite unpredictable. Google don't provide any reliable way to do this so I've had  to "hack" the display to get this to work at all so it's very unlikely I could fix this if it's caused by non-default display scaling. Please do let me know if using 100% fixes the issue though, and if it does I'll see if there's anything I can do. If is doesn't, do you have Chrome/Brave zoom enabled or any other Chrome Extensions installed?
